perlmod/Fink/Bootstrap.pm

```diff
@@ -245,6 +245,12 @@ GCC_MSG
 			"of Mac OS X might work with Fink, but there are no " .
 			"guarantees.");
 		$distribution = "10.13";
+	} elsif ($host =~ /^i386-apple-darwin(\d+)\./) {
```

Comment:
This should probably be a pair of "darwin18" checks, not a duplicate of the next block that is the fallback for unknown kernel numbers.

Reply:
You are right @dmacks. There should indeed be a pair of darwin18 checks as above.

Reply:
Addressed in PR #170